### PR TITLE
fix: when client close the connection, no error should occured

### PR DESCRIPTION
**Pull request by fanweixiao**
Related to #296 

- bump `y3` to `v1.0.5`, will not suppress io.EOF error
- client won't treat ApplicationError(0x00) as error
- server won't treat ApplicationError(0x00) as error
- need update to `v1.6.1` as bug fixed

---
This change is [Reviewable](https://reviewable.io/reviews/yomorun/yomo/297)

### CODE REVIEW

File: example/0-basic/Taskfile.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ tasks:
     cmds:
       - "./bin/source{{exeExt}}"
     env:
-      YOMO_LOG_LEVEL: error
+      YOMO_LOG_LEVEL: debug
```
Comment (venjiang) [inline example/0-basic/Taskfile.yml:42]: need debug mode？

File: core/server.go [hunk 2]
```diff
@@ -116,14 +113,14 @@ func (s *Server) Serve(ctx context.Context, conn net.PacketConn) error {
...
+					// @CC: when Source close the connection, it won't affect connectors
 					app, ok := s.connector.App(connID)
 					if ok {
 						// connector
 						s.connector.Remove(connID)
 						// store
 						// when remove store by appID? let me think...
-						logger.Errorf("%s❤️3/ [%s::%s](%s) on stream %v", ServerLogPrefix, app.ID(), app.Name(), connID, err)
-						logger.Printf("%s💔 [%s::%s](%s) is disconnected", ServerLogPrefix, app.ID(), app.Name(), connID)
+						logger.Infof("%s💔 [%s::%s](%s) close the connection", ServerLogPrefix, app.ID(), app.Name(), connID)
```
Comment (venjiang) [inline core/server.go:123]: suggest using logger.Printf

File: core/client.go [hunk 3]
```diff
@@ -194,15 +203,15 @@ func (c *Client) handleFrame() {
 
 // Close the client.
 func (c *Client) Close() (err error) {
-	c.logger.Debugf("%sclose the connection", ClientLogPrefix)
+	c.logger.Infof("%sclose the connection, name:%s, addr:%s", ClientLogPrefix, c.name, c.session.RemoteAddr().String())
```
Comment (venjiang) [inline core/client.go:206]: suggest using logger.Printf

File: example/0-basic/Taskfile.yml (hunk 1 above)
Comment (fanweixiao) [reply example/0-basic/Taskfile.yml:42]: yes, should error

**Review by xiaojian-hong — APPROVED** [review]
👍

**Pull request merged**